RaveAddIn/XML/vbet.xml

```diff
@@ -35,7 +35,7 @@
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_90']" type="vector" symbology="" />
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_100']" type="vector" symbology="" />
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='VBET_NETWORK']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Raster[@id='EVIDENCE']" type="raster" symbology="" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Raster[@id='EVIDENCE']" type="raster" symbology="" transparency="40"/>
```

Comment:
Make sure these VBET changes don't collide with @Cashe93 changes to VBET. Some overlap.

Reply:
@Cashe93 I think we can avoid this as long as you add transparencies when you merge your branch from issue #68 .